oldlog.go

```diff
@@ -97,6 +97,19 @@ func SetLogLevel(name, level string) error {
 	return nil
 }
 
+// GetSubsystems returns a slice containing the
+// names of the current loggers
+func GetSubsystems() []string {
+	subs := make([]string, len(loggers))
+
+	i := 0
+	for k := range loggers {
```

Comment:
the boundary checking is the same either way, doing `subs[i] = k` requires a boundary check as well.